awx/api/views/__init__.py

```diff
@@ -2509,7 +2509,7 @@ def _validate_spec_data(new_spec, old_spec):
                 ).format(
                     allowed_types=', '.join(JobTemplateSurveySpec.ALLOWED_TYPES.keys()), **context
                 )))
-            if 'default' in survey_item:
+            if 'default' in survey_item and survey_item['default'] != u'':
```

Comment:
Don't need `u` literals in this diff, this is python3 😄

Reply:
my bad, thought we were maintaining Py2 compatibility :/